src/components/StoryCreationComponent.js

```diff
@@ -68,78 +52,122 @@ class StoryCreationComponent extends Component {
   };
 
   handleChangeTags = event => {
-    console.log(this.state.tagsField);
     let tag = event.target.value.toLowerCase();
     tag = tag.replace('\\', '');
```

Comment:
We should prolly only allow letters/numbers/hyphen for tags

Reply:
What if the user want's to add dollar signs, percents, quotes? I'm on the fence, but I see the benefit of filtering out non alphanumeric symbols

Reply:
I'm not convinced we need those symbols in tags. Being used as a mechanism for story searching, it might introduce complication if special characters are allowed.
On the flip side, I think white space should be allowed in a tag. An example article from Medium, which shows tags with white spaces at the bottom: https://medium.com/s/story/the-democratic-presidential-primary-needs-the-democratic-party-73f90c006348

Reply:
I brought this up before in slack...twice actually. I should be home from work by the time our meeting starts tonight (4pm pst). I'll bring it up again then.